client/boot/index.js

```diff
@@ -52,6 +53,9 @@ function init() {
 
 	debug( 'Starting Calypso. Let\'s do this.' );
 
+	// prune sync-handler records
+	syncHandler.pruneRecordsFrom();
```

Comment:
> I too have wanted to bring in some time constants, but I prefer basic constants instead of a library to produce them.

I don't see what is the problem to use a library for that? For instance we are using `lodash/noop` instead of arrow `() => {}` function. In this sense it's the same use case.

Personally I'm pretty confident that this code ...

```es6
cache.pruneRecordsFrom( '2 days' );
```

... is clearer, simpler and easy to follow than this one:

```es6
const DAY_IN_HOURS = 24;
const HOUR_IN_MS = 3600000;
const LIFETIME = 2 * DAY_IN_HOURS * HOUR_IN_MS;
cache.pruneRecordsFrom(  LIFETIME );
```

Again it's just my personal opinion. I'll very glad to use whatever we choose. :sunglasses: